smart-contracts/contracts/dex-router-osmosis/src/msg.rs

```diff
@@ -0,0 +1,68 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Coin, Uint128};
+use osmosis_std::types::osmosis::poolmanager::v1beta1::SwapAmountInRoute;
+
+#[cw_serde]
+pub struct InstantiateMsg {}
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    Swap {
+        out_denom: String,
+        path: Option<Vec<SwapAmountInRoute>>,
+        minimum_receive: Option<Uint128>,
+        to: Option<String>,
+    },
+    SetPath {
+        offer_denom: String,
+        ask_denom: String,
+        path: Vec<u64>,
+        bidirectional: bool,
+    },
+    RemovePath {
+        offer_denom: String,
+        ask_denom: String,
+        path: Vec<u64>,
+        bidirectional: bool,
```

Comment:
Can this field be removed? I think we only need the index selectors here during the remove.

Reply:
Yes. I have it here for symmetry. Both in SetPath and RemovePath we could leave it out. As it is in SetPath, the interface looks a bit more consistent to me with having it here too. I can remove it if you prefer to, else I will keep it.

Reply:
If it is useful we can keep it. If not we should remove